### DISCUSSION TITLE
I'm not able to display my website

### DISCUSSION THREAD

**Discussion by Sarveshdhumal** [community/community#23685] [Other Feature Feedback, Questions, & Ideas]
<p>I am trying to upload my files but they aren’t showing up-</p>
<p>here are my files: <a href="https://github.com/Sarveshdhumal/dicegame" rel="noopener nofollow ugc">https://github.com/Sarveshdhumal/dicegame</a></p>
<p>And here is link: <a href="https://sarveshdhumal.github.io/dicegame/" rel="noopener nofollow ugc">https://sarveshdhumal.github.io/dicegame/</a></p>

**Comment by thinkverse** [comment, marked as answer]
<p>Hi there.  :wave:</p>
<p>Your site will be available at <a href="https://sarveshdhumal.github.io/dicegame/Dice%20game/dice.html" rel="noopener nofollow ugc">/dicegame/Dice%20game/dice.html</a>, the reason being is GitHub Pages works from the root directory. With simple HTML sites, by default Pages will look for <code>index.html</code>, if that’s not found it falls back to the <code>README.md</code> file.</p>
<p>To get your page to be displayed at <a href="https://sarveshdhumal.github.io/dicegame" rel="noopener nofollow ugc">/dicegame</a>, I would move the files out of the <code>Dice game</code> folder into the root and rename the <code>dice.html</code> to <code>index.html</code>.</p>
<p>Hope that helps. :slightly_smiling_face:</p>
<p>The <code>%20</code> is the encoding for the space in the folder name FYI if you were wondering about that.</p>

**Comment by Sarveshdhumal** [comment]
<p>Thank you. It was really helpful:)</p>
